index.d.ts

```diff
@@ -0,0 +1,63 @@
+/**
```

Comment:
Added function descriptions, examples and types for typescript coders (Declaration File - index.d.ts)